aptos-move/block-executor/src/executor.rs

```diff
@@ -367,12 +339,13 @@ where
         }
     }
 
-    fn worker_commit_hook(
+    fn commit_epilogue(
```

Comment:
epilogue means something else in VM, and this doesn't refer to that - so it is a big confusing.

maybe calling it commit_materialization / commit_materialize_results / commit_finalization / commit_compute_final_results ?

Reply:
there's a hidden assumption that this `commit_epilogue` can be executed out of order, we probably want to call it out explicitly